TestCases/turbomachinery/axial_stage_BFM/config.cfg

```diff
@@ -0,0 +1,268 @@
+MATH_PROBLEM= DIRECT
+%
+% Restart solution (NO, YES)
+RESTART_SOL= NO
+%
+%
+% -------------------- COMPRESSIBLE FREE-STREAM DEFINITION --------------------%
+%
+% Mach number (non-dimensional, based on the free-stream values)
+MACH_NUMBER= 0.2
+%
+% Free-stream Turbulence Intensity
+FREESTREAM_TURBULENCEINTENSITY = 0.03
+%
+% Free-stream Turbulent to Laminar viscosity ratio
+FREESTREAM_TURB2LAMVISCRATIO = 100.0
+%
+%Init option to choose between Reynolds (default) or thermodynamics quantities for initializing the solution (REYNOLDS, TD_CONDITIONS)
+INIT_OPTION= TD_CONDITIONS
+%
+% ---------------------- REFERENCE VALUE DEFINITION ---------------------------%
+%
+% Reference origin for moment computation
+REF_ORIGIN_MOMENT_X = 0.00
+REF_ORIGIN_MOMENT_Y = 0.00
+REF_ORIGIN_MOMENT_Z = 0.00
+%
+% Reference length for pitching, rolling, and yawing non-dimensional moment
+REF_LENGTH= 1.0
+%
+% Reference area for force coefficients (0 implies automatic calculation)
+REF_AREA= 1.0
+%
+%
+% Flow non-dimensionalization (DIMENSIONAL, FREESTREAM_PRESS_EQ_ONE,
+%                              FREESTREAM_VEL_EQ_MACH, FREESTREAM_VEL_EQ_ONE)
+REF_DIMENSIONALIZATION= DIMENSIONAL
+MOTION_ORIGIN = 0 0 0
+FREESTREAM_VELOCITY= ( 0.0, 0.00, 1.00 )
+%
+%
+% ------------------------------ EQUATION OF STATE ----------------------------%
+%
+% Different gas model (STANDARD_AIR, IDEAL_GAS, VW_GAS, PR_GAS)
+FLUID_MODEL= IDEAL_GAS
+%
+% Ratio of specific heats (1.4 default and the value is hardcoded
+%                          for the model STANDARD_AIR)
+GAMMA_VALUE= 1.4
+%
+% Specific gas constant (287.058 J/kg*K default and this value is hardcoded 
+%                        for the model STANDARD_AIR)
+GAS_CONSTANT= 287.058
+%
+% Critical Temperature (273.15 K by default)
+CRITICAL_TEMPERATURE= 273.15
+%
+% Critical Pressure (101325.0 N/m^2 by default)
+CRITICAL_PRESSURE= 101325
+%
+% Acentri factor (0.035 (air))
+ACENTRIC_FACTOR= 0.035
+%
+%
+% ----------------------- BODY FORCE DEFINITION -------------------------------%
+%
+% Apply a body force as a source term (NO, YES)
+BODY_FORCE= YES
+
+% Select what type of body force to use (CONSTANT_BF, VARIABLE_BF)
+BODY_FORCE_TYPE= VARIABLE_BF
+
+BFM_INPUT_FILENAME= blade_geometry.drg
+
+% Determine rotation for body force calculation in RPM (For 2D, negative if moving in the negative y-direction)
+BODY_FORCE_ROTATION= 0
+
+BFM_FORMULATION= THOLLET
+
+BFM_ROTATION_AXIS = 0.0 0.0 1.0
+% --------------------------- VISCOSITY MODEL ---------------------------------%
+%
+% Viscosity model (SUTHERLAND, CONSTANT_VISCOSITY).
+VISCOSITY_MODEL= CONSTANT_VISCOSITY
+%
+% Molecular Viscosity that would be constant (1.716E-5 by default)
+MU_CONSTANT= 1.3764E-5
+%
+% Sutherland Viscosity Ref (1.716E-5 default value for AIR SI)
+MU_REF= 1.716E-5
+%
+% Sutherland Temperature Ref (273.15 K default value for AIR SI)
+MU_T_REF= 273.15
+%
+% Sutherland constant (110.4 default value for AIR SI)
+SUTHERLAND_CONSTANT= 110.4
```

Comment:
Remove these comments that are already in the config template.
And any option you are not using, e.g. JST_SENSOR_COEFF or that is set to default.